@stellar/anchor-tests/src/types.ts

```diff
@@ -106,6 +107,89 @@ export interface Sep12Config {
   sameAccountDifferentMemos?: [string, string];
 }
 
+/**
+ * The configuration object for SEP-24 tests.
+ */
+export interface Sep24Config {
+  /**
+   * The credentials that should be used to fetch the pending and completed transactions listed below
+   * (depositPendingTransaction, depositCompletedTransaction, withdrawPendingUserTransferStartTransaction
+   * and withdrawCompletedTransaction).
+   *
+   * - accountAddress: the public key of the account which holds the pending and completed transactions.
+   * - accountSignerSecretKey: the secret key of a valid signer of the accountAddress.
+   *
+   * When provided, it'll be used in the sep10 "returns a valid JWT" test.
+   */
+  accountHolder: {
+    accountAddress: string;
+    accountSignerSecretKey: string;
+  };
```

Comment:
SEP-31's config object has something similar, `sendingAnchorClientSecret`. We don't actually need the public key configured right? Could we do something like `authAccountSecret`?

Reply:
So in most cases we will only need the secret key as we can infer the public key from it. But in certain cases (like Vibrant accounts) we have a public key with weigh zero so we need a separate secret key to actually sign stuff. We can probably make the `accountAddress` optional and in case it's not provided then we infer the public key using the provided secret key. How does that sound?

About the naming, I thought about using `accountSigner...` to make it clear we are talking about a signer of the account in question. But, as in most cases Anchors will only need to pass the secretKey maybe we should make it less confusing for the common case.

What do you think about renaming the whole object like the below ?
```
account: {
    publicKey?: string;
    secretKey: string;
  };
```

Reply:
I like that approach!

Reply:
Done!